Soneta.Platform.Developer/WorkerItemTemplate/Page.pageform.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:xsd="http://www.w3.org/2001/XMLSchema" 
           xmlns="http://www.enova.pl/schema/form.xsd" 
           Priority="$worker_priority$" 
-          DataType="$rootnamespace$.$worker_class$,$defaultnamespace$">
+          DataType="$rootnamespace$.$worker_paramsclass$,$rootnamespace$">
   <Page Name="$worker_class$Page" CaptionHtml="$worker_caption$" DataContext="{DataSource}">
```

Comment:
Idąc za ciosem to jeszcze atrybuty Name i CaptionHtml teraz nie konsekwentnie nadal wskazują na worker, a nie na parametry.
Okno dla workera, to może być inne i dodatkowe i pewnie nie ma sensu mieszać parametrów i workera na oknach, które są im dedykowane indywidualnie. Ten template nie przewiduje okna dla workera.

Reply:
Zgadzam się i poprawiłem zgodnie z zaleceniami. Uporządkowałem też nieco nazwy parametrów dla worker'a, żeby wyglądały spójniej.